ledger-tool/src/program.rs

```diff
@@ -131,18 +131,52 @@ fn load_blockstore(ledger_path: &Path, arg_matches: &ArgMatches<'_>) -> Arc<Bank
     bank
 }
 
-pub trait RunSubCommand {
-    fn run_subcommand(self) -> Self;
+pub trait ProgramSubCommand {
+    fn program_subcommand(self) -> Self;
 }
 
-impl RunSubCommand for App<'_, '_> {
-    fn run_subcommand(self) -> Self {
+impl ProgramSubCommand for App<'_, '_> {
+    fn program_subcommand(self) -> Self {
+        let program_arg = Arg::with_name("PROGRAM")
+            .help(
+                "Program file to use. This is either an ELF shared-object file to be executed, \
+                 or an assembly file to be assembled and executed.",
+            )
+            .required(true)
+            .index(1);
         self.subcommand(
+            SubCommand::with_name("program")
+        .about("Run to test, debug, and analyze on-chain programs.")
+        .subcommand(
+            SubCommand::with_name("cfg")
+                .about("generates Control Flow Graph of the program.")
+                .arg(
+                    Arg::with_name("max_genesis_archive_unpacked_size")
+                        .long("max-genesis-archive-unpacked-size")
+                        .value_name("NUMBER")
+                        .takes_value(true)
+                        .default_value("10485760")
+                        .help("maximum total uncompressed size of unpacked genesis archive")
+                )
+                .arg(program_arg.clone())
```

Comment:
nit: Does this have to be cloned? In `main.rs`, we pass some args by ref such as here:
https://github.com/solana-labs/solana/blob/5377bd6de4c2b60af6a84ddeb55ef2c5256dba34/ledger-tool/src/main.rs#L1325

Reply:
changed to references.